docs/devops-guide/manual.md

```diff
@@ -4,7 +4,12 @@ title: DevOps Guide (manual install)
 sidebar_label: Manual installation
 ---
 
-**WARNING:** Manual installation is not recommended. We recommend following the [quick-install](devops-guide-quickstart) document. The current document describes the steps that are needed to install a working deployment, but steps are easy to mess up, and the debian packages are more up-to-date, where this document is sometimes not updated to reflect latest changes.
+**WARNING:**
```

Comment:
I added "admonition" support so you can write it like this:

```
:::warning Manual installation is not recommended
bla bla bla
:::
```

And it will be rendered like the blue square here: https://jitsi.github.io/handbook/docs/intro#about-this-handbook but red.